src/styles/product-cart.scss

```diff
@@ -0,0 +1,140 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  transition: transform 300ms ease;
+
+  &:hover {
+    transform: scale(1.1);
+    cursor: pointer;
```

Comment:
Why do you change the cursor on hover?
If you want to have a cursor pointer just set it to `.card` :)